templates/taxi/car_list.html

```diff
@@ -1,17 +1,38 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
   {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
+    <table class="table">
+    <tr>
+      <th>ID</th>
+      <th>Model</th>
+      <th>Manufacturer</th>
+      <th>Update</th>
+      <th>Delete</th>
+    </tr>
+    {% for car in car_list %}
+      <tr>
+        <td>
+          {{ car.id }}
+        </td>
+        <td>
+          {{ car.model }}
+        </td>
+        <td>
+          {{ car.manufacturer }}
+        </td>
+        <td>
+          <a href="{% url 'taxi:car-update' pk=car.id %}">Update</a>
+        </td>
+        <td>
+          <a href="{% url 'taxi:car-delete' pk=car.id %}">Delete</a>
```

Comment:
Consider adding a 'Cancel' button next to the 'Delete' link to allow users to navigate back to the previous page, as per the checklist requirement: 'Add `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.'